src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -72,18 +72,24 @@ private IVariableDeclaration CreateVariableDeclarationInternal(BoundLocalDeclara
                 {
                     // There is no explicit syntax for the initializer, so we use the initializerValue's syntax and mark the operation as implicit.
                     initializerSyntax = initializerValue.Syntax;
-                    isImplicit = true;
+                    initializerIsImplicit = true;
                 }
 
-                initializer = OperationFactory.CreateVariableInitializer(initializerSyntax, initializerValue, _semanticModel, isImplicit);
+                initializer = OperationFactory.CreateVariableInitializer(initializerSyntax, initializerValue, _semanticModel, initializerIsImplicit);
             }
 
-            return OperationFactory.CreateVariableDeclaration(boundLocalDeclaration.LocalSymbol, initializer, _semanticModel, syntax);
+            ILocalSymbol symbol = boundLocalDeclaration.LocalSymbol;
+            SyntaxNode syntaxNode = boundLocalDeclaration.Syntax;
+            ITypeSymbol type = null;
+            Optional<object> constantValue = default;
+            bool isImplicit = false;
+
+            return new SingleVariableDeclaration(symbol, initializer, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
         private IVariableDeclaration CreateVariableDeclaration(BoundLocal boundLocal)
```

Comment:
>IVariableDeclaration [](start = 16, length = 20)

ISingleVariableDeclaration?

Reply:
Done.

---
In reply to: [146939280](https://github.com/dotnet/roslyn/pull/22842#discussion_r146939280) [](ancestors = 146939280)